chainer/training/extensions/log_report.py

```diff
@@ -54,10 +57,14 @@ class LogReport(extension.Extension):
 
     """
 
-    def __init__(self, keys=None, trigger=(1, 'epoch'), postprocess=None,
-                 log_name='log'):
+    def __init__(self, keys=None, trigger=(1, 'epoch'), write_trigger=None,
+                 postprocess=None, log_name='log'):
```

Comment:
I noticed that adding `write_trigger` option in the middle breaks users' code. Although we can break compatibility theoretically (cf. [our compatibility policy](https://docs.chainer.org/en/stable/compatibility.html#breaking-the-compatibility)), we need not unnecessarily do that if we can avoid it, thought the order of options is a bit natural. Could you add the option to the last position?